app/components/primary_navigation_component.html.erb

```diff
@@ -1,19 +1,15 @@
 <div class="app-primary-navigation govuk-!-display-none-print">
-  <div class="app-primary-navigation__container">
-    <div class="app-primary-navigation__nav">
-      <nav class="app-primary-navigation" aria-label="main menu">
-        <ul class="app-primary-navigation__list">
-          <% @navigation_items.each do |nav_item| %>
-            <li class="app-primary-navigation__item <%= nav_item.classes %>">
-              <a class="app-primary-navigation__link"
-                <%= nav_item.active ? 'aria-current=page' : '' %>
-                 href="<%= nav_item.href %>">
-                 <%= nav_item.text %>
-              </a>
-            </li>
-          <% end %>
-        </ul>
-      </nav>
-    </div>
+  <div class="govuk-width-container">
```

Comment:
Should this have `app-primary-navigation` on it?

Reply:
`app-primary-navigation` is the parent `div` that stretches full width and adds the background colour.
`govuk-width-container` fixes the width of the content within this `div`.

Each thing does one thing, and this follows the pattern of other common components, like the header and footer.

Reply:
Sorry - I didn't see the parent in the code snippet.